### ISSUE TITLE
Integration support with IDE, please

### ISSUE THREAD

**Issue by migueltk** [TheThirdOne/rars#9]
Normally I program in C# with the monodevelop IDE, I have also worked with the IDE netbeans, something basic, it took me a while to create the project in netbeans and execute it. 
Which IDE do you use for the development of RARS? Would it be possible in the future to support an IDE to make everything simpler? open the IDE, go to the RARS folder, load the project and everything is ready.
Thanks for the project ... it's great!

**Comment by TheThirdOne** [comment]
I have used IntelliJ to do most of my work on Rars, but don't want to impose a decision of IDE onto the project. It should be quite simple to import the project into most IDEs.

Did your difficulty come from not knowing how the project was set up (such as where the main class is) or from difficulty actually importing the project?

**Comment by TheThirdOne** [comment]
Ideally it shouldn't throw a null pointer exception with invalid property files. I will look into some better error handling for that to give a better error message. 

As for IDE status, I won't include any configuration files into the repo, but if you want to you could add a wiki page or section to the readme explaining how to set up the project in an IDE.

I may make a build file of some kind (likely make or sh) to build jars though. Building jars can definitely be a little hard to set up and it would be good to have something reproducible for releases.

**Comment by TheThirdOne** [comment]
In 694f9e3bf57c7bd279076cb66c483d3cf0f1f748, I did some work to minimize issues with missing properties files. The only properties file that is neccessary is syscall.properties and an error message is logged with the general solution to that. 

Additionally, I added a [build script]( https://github.com/TheThirdOne/rars/blob/efdaea9335a07eb88acbfcd33013712738adf9ad/build-jar.sh). It probably won't work on windows (but it might through the linux subsystem), but it should be a consistent way to get a working build without an IDE. 

In the future, I would like to move it over to Make, Gradle or Maven, but it was easy to use sh.
